src/MCPClient/tests/test_create_mets_v2.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+import sys
+
+THIS_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.abspath(os.path.join(THIS_DIR, "../lib/clientScripts")))
+
+from create_mets_v2 import createDMDIDsFromCSVMetadata
+
+
+def test_createDMDIDsFromCSVMetadata_finds_non_ascii_paths(mocker):
+    dmd_secs_creator_mock = mocker.patch(
+        "create_mets_v2.createDmdSecsFromCSVParsedMetadata", return_value=[]
+    )
+    state_mock = mocker.Mock(
+        **{
+            "CSV_METADATA": {
+                "montréal".encode("utf8"): "montreal metadata",
+                "dvořák".encode("utf8"): "dvorak metadata",
+            }
+        }
+    )
```

Comment:
As ever your tests are great to read and I learn a lot from CR-ing this type of thing. Nice work.